apps/ui/src/views/Space/Settings.vue

```diff
@@ -527,12 +526,9 @@ watchEffect(() => setTitle(`Edit settings - ${props.space.name}`));
         @update-validity="v => (hasAdvancedErrors = !v)"
       />
     </UiContainerSettings>
-    <div
-      v-if="
-        !uiStore.sidebarOpen &&
-        ((isModified && canModifySettings && !hasAdvancedErrors) || error)
-      "
-      class="fixed bg-skin-bg bottom-0 left-0 right-0 lg:left-[312px] xl:right-[240px] border-y px-4 py-3 flex flex-col xs:flex-row justify-between items-center"
+    <UiToolbarBottom
+      v-if="(isModified && canModifySettings) || error"
```

Comment:
Blocking this PR until it's clarified if it was intended.

Reply:
No, it's not intended, may due to a merge conflict gone wrong